src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/media/VideoComponentObject.java

```diff
@@ -35,44 +35,44 @@ public WebElement getVideoPlayerIframe() {
 
   public void verifyVideoEmbedWidth() {
     Assertion.assertEquals(videoEmbed.findElement(By.tagName("div")).getCssValue("width"),
-                           videoWidth + "px");
-    PageObjectLogging.log("verifyVideoEmbedWidth", "Width: " + videoWidth, true);
+        videoWidth + "px");
+    LOG.success("verifyVideoEmbedWidth", "Width: " + videoWidth);
   }
 
   public void verifyVideoIframeWidth() {
     Assertion.assertEquals(videoEmbed.findElement(By.tagName("iframe")).getAttribute("width"),
-                           videoWidth.toString());
-    PageObjectLogging.log("verifyVideoIframeWidth", "Width: " + videoWidth, true);
+        videoWidth.toString());
+    LOG.success("verifyVideoIframeWidth", "Width: " + videoWidth);
   }
 
   public void verifyVideoOoyalaAgeGate() {
     String ageGateClass = "ageGate";
     Assertion.assertTrue(videoEmbed.findElement(By.className(ageGateClass)) != null);
-    PageObjectLogging.log("verifyVideoOoyalaAgeGate", "Age gate module is enabled", true);
+    LOG.success("verifyVideoOoyalaAgeGate", "Age gate module is enabled");
   }
 
   public void verifyVideoOoyalaEmbed() {
     WebElement container = videoEmbed.findElement(By.tagName("div"));
     String containerId = "ooyalaplayer-";
     Assertion.assertStringContains(container.getAttribute("id"), containerId);
     wait.forElementVisible(container.findElement(By.tagName("object")));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "object" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)